src/components/home/about/imageList.js

```diff
@@ -0,0 +1,29 @@
+// need to replace with proper photos
+export const imageList = [
+  {
+    image:
+      'https://assets.rebelmouse.io/eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpbWFnZSI6Imh0dHBzOi8vYXNzZXRzLnJibC5tcy8zMzI0NzU5Ni9vcmlnaW4uanBnIiwiZXhwaXJlc19hdCI6MTczNzI1Nzg5MH0.PrIBXmpEeLrVe--JDq6kE2HF-DPuyzaLgfC1tnQ10Eo/img.jpg?width=742&quality=80',
+    alt: 'Volunteer image one'
```

Comment:
When the images are updated, the `alt`s may have to be a bit more descriptive than this. At least that's my understanding of accessibility.

Reply:
Agreed. I didn't change these as I don't know how to describe the images without knowing what the images are.

Reply:
they will be images of a person so its probably sufficient to say `volunteer headshot image 1 out of 5` or `volunteer headshot image of Hugh` ect.. These will also end up getting added to the rotating images as alt text that gets mapped in eventually.